docs/ERC-725.md

```diff
@@ -69,52 +78,54 @@ Others may be added in the future.
 
 #### Executed
 
-MUST be triggered when `execute` creates a new call using any `operationType`.
-
 ```solidity
 event Executed(uint256 indexed _operation, address indexed _to, uint256 indexed _value, bytes _data);
 ``` 
+MUST be triggered when `execute` creates a new call using any `operationType`.
```

Comment:
actually besides 1,2, as the data would also be far to big for a log to expensive.